src/javascript.rs

```diff
@@ -111,6 +118,40 @@ fn module_function<'a>(
     ])
 }
 
+fn external_function<'a, T>(
+    public: bool,
+    name: &'a str,
+    arguments: &'a [ExternalFnArg<T>],
+    module: &'a str,
+    fun: &'a str,
+) -> Output<'a> {
+    let head = if public {
+        "export function "
+    } else {
+        "function "
+    };
+    let arguments = wrap_args(arguments.iter().enumerate().map(|a| match a {
+        (index, ExternalFnArg { label, .. }) => {
+            let arg_name = label.clone().unwrap_or(format!("arg{}", index));
+            Document::String(arg_name)
+        }
+    }));
+    let body = module
+        .to_doc()
+        .append(".")
+        .append(fun)
+        .append(arguments.clone());
```

Comment:
Need a `return` too. Pesky statement based language.
```suggestion
    let body = docvec!["return ", module, ".", fun, arguments.clone()];
```